test/integration/image-component/default/components/TallImage.js

```diff
@@ -1,20 +0,0 @@
-import React from 'react'
-import Image from 'next/image'
-
-import testTall from './tall.png'
-
-const Page = () => {
-  return (
-    <div>
-      <h1 id="page-header">Static Image</h1>
-      <Image
-        id="basic-static"
-        src={testTall}
-        layout="fixed"
-        placeholder="blur"
-      />
-    </div>
-  )
-}
-
-export default Page
```

Comment:
Why was this test deleted?

Reply:
Ah, thats why it was a seperate component 🤔 will revert